src/Detail.tsx

```diff
@@ -94,47 +94,46 @@ export const Detail = (props: DetailProps) => {
       setAnnotations(broccoli.anno);
       setViews(broccoli.views);
 
-      if (params.tier2) {
-        let annoToZoom: AnnoRepoAnnotation[];
-        if (params.tier2.includes("resolution")) {
```

Comment:
Make zoomAnno function configurable and make annotation type configurable in project config.